apps/www/app/(routes)/methods/page.tsx

```
@@ -1,5 +1,122 @@
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger,
+} from "@/components/ui/accordion"
+import { Card } from "@/components/ui/card"
+import { siteConfig } from "@/config/site.config"
+import Image from "next/image"
+import { useRef, useEffect, useState } from "react"
 
-export default function Methods() {
-  return <div>Fuck</div>;
+type AccordionItem = {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'AccordionItem'. Consider to delete it or rename it.

'AccordionItem' is defined here:


(lint/suspicious/noRedeclare)